mail/api.py

```diff
@@ -18,14 +18,23 @@ class MailgunClient:
     _basic_auth_credentials = ('api', settings.MAILGUN_KEY)
 
     @staticmethod
-    def get_base_params():
+    def get_base_params(sender_name=None):
         """
         Base params for Mailgun request. This a method instead of an attribute to allow for overrides.
+
+        Args:
+            sender_name (str): email's sender name.
+
+        Returns:
+            json: json object containing email from info.
         """
-        return {'from': settings.MAILGUN_FROM_EMAIL}
+        email_from = settings.MAILGUN_FROM_EMAIL
+        if sender_name:
+            email_from = "{sender_name} <{email}>".format(sender_name=sender_name, email=email_from)
+        return {'from': email_from}
```

Comment:
Looking at @pdpinch example, I think the `sender_name` is not `settings.MAILGUN_FROM_EMAIL`